docs/reference/technology/mesh/quick_start_intro.md

```diff
@@ -57,13 +57,3 @@ In a mesh/tree network, all nodes are configured as 6LoWPAN routers.
 In a 6LoWPAN network, RPL protocol is used for forming the routing topology. Every node selects a primary parent for routing so the result looks like a tree.
```

Comment:
Query: Should line 39 also change?

Reply:
yes it should

Reply:
I updated the line #39 and pushed a rebased version in. Would you please recheck?